src/calendar/CalendarMonth.jsx

```diff
@@ -32,6 +26,12 @@ const CalendarMonth = React.createClass({
     value: CustomPropTypes.momentOrMomentRange,
   },
 
+  componentWillMount() {
+    const lang = moment().localeData();
```

Comment:
Ahh, I see. So changing the `locale` doesn't itself trigger a re-render? I think incorporating #56's approach is probably a good idea. You have any thoughts on this, @AlanFoster ?